src/ch/imedias/rsccfx/RsccApp.java

```diff
@@ -21,18 +24,33 @@ public static void main(String[] args) {
   @Override
   public void start(Stage stage) {
 
-    Rscc model = new Rscc(new SystemCommander());
-    RsccView view = new RsccView(model);
+    model = new Rscc(new SystemCommander());
+    // FIXME: change appropriately to what should be shown and reflect changes in other views / presenters
```

Comment:
change appropriately to what should be shown and reflect changes in other views / presenters